sklearn/cluster/_hierarchical.py

```diff
@@ -960,8 +960,11 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         be advantageous to compute the full tree. It must be ``True`` if
         ``distance_threshold`` is not ``None``.
 
-    linkage : {"ward", "complete", "average", "single"}, optional\
-            (default="ward")
+        - auto by default takes True if distance_threshold is not None,
```

Comment:
No need for bullet point and I think this would be clearer:


```
By default `compute_full_tree` is "auto", which is equivalent to
`True` when `distance_threshold` is not `None` or that
`n_clusters` is inferior to 100 or `0.02 * n_samples`. Otherwise,
"auto" is equivalent to `False`.
```